components/email.php

```diff
@@ -0,0 +1,213 @@
+<?php
+/**
+ * Manage BuddyPress Email Post Types.
+ *
+ * @since 1.6.0
+ */
+class BPCLI_Email extends BPCLI_Component {
+	/**
+	 * Create a new email post connected to an email type.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--type]
+	 * : Required. Email type for the email (should be unique identifier, sanitized like a post slug).
+	 *
+	 * [--type_description]
```

Comment:
We are sticking to a pattern in using - instead of underscores. So `--type-description` is correct.

Reply:
I copied how WP CLI handles this as they use underscores.  See `wp post create`:
https://developer.wordpress.org/cli/commands/post/

If we are going with a hyphen, then that's good to know.  Will make the change.

Reply:
Yes! I and Boone made a decision, in the beginning, to use hyphen instead of underscore,